contrib/container-chroot-agent/Dockerfile-cfengine-chroot-agent

```diff
@@ -0,0 +1,14 @@
+#+caption: Dockerfile for CFEngine Enterprise Agent
+#+begin_src
+FROM debian:11
+RUN apt-get update && \
```

Comment:
come to think of it, I was only installed/managing locales to enable installing easier, will try removing this entirely.